devtools/cli/src/test/java/io/quarkus/cli/MavenProjectInfoAndUpdateTest.java

```diff
@@ -180,10 +181,10 @@ private static void assertQuarkusPlatformBoms(String output, String... coords) {
     private static void assertQuarkusPlatformBomUpdates(String output, ArtifactCoords... coords) {
         final StringWriter buf = new StringWriter();
         try (BufferedWriter writer = new BufferedWriter(buf)) {
-            writer.write("[INFO] Recommended Quarkus platform BOM updates:");
+            writer.write("[INFO] Suggested Quarkus platform BOM(s) updates:");
```

Comment:
```suggestion
            writer.write("[INFO] Suggested Quarkus platform BOM updates:");
```
"BOM updates" is already plural. If there was a single BOM update it would be simply "BOM update"